packages/ai-bot/helpers.ts

```diff
@@ -768,3 +768,34 @@ export function isCommandResultEvent(
       APP_BOXEL_COMMAND_RESULT_REL_TYPE
   );
 }
+
+function elideCodeBlocks(content: string) {
+  // Define constants for marker texts
+  const SEARCH_MARKER: string = '<<<<<<< SEARCH';
```

Comment:
We already have these constants in `host/app/lib/search-replace-block-parsing.ts`, maybe we can extract this lib to runtime-common